assets/inference/environments/mlflow-ubuntu22.04-py312-cpu-inference/tests/mlflow_test.py

```diff
@@ -28,27 +28,30 @@ def test_minimal_cpu_inference():
         AzureCliCredential(), subscription_id, resource_group, workspace_name
     )
 
-    env_name = "minimal_cpu_inference"
+    env_name = "mlflow_py312_inference"
 
     env_docker_context = Environment(
         build=BuildContext(path=this_dir / BUILD_CONTEXT),
-        name="minimal_cpu_inference",
-        description="minimal 22.04 py312 cpu inference environment created from a Docker context.",
+        name="mlflow_py312_inference",
+        description="mlflow 22.04 py312 cpu inference environment created from a Docker context.",
     )
     ml_client.environments.create_or_update(env_docker_context)
 
     # create the command
     job = command(
         code=this_dir / JOB_SOURCE_CODE,  # local path where the code is stored
-        command="python main.py --score ${{inputs.score}}",
+        command="python main.py --model_dir ${{inputs.model_dir}} "
+        "--score ${{inputs.score}} --score_input ${{inputs.score_input}}",
         inputs=dict(
-            score="valid_score.py",
+            score="/var/mlflow_resources/mlflow_score_script.py",
+            score_input="sample_2_0_input.txt",
+            model_dir="mlflow_2_0_model_folder"
         ),
         environment=f"{env_name}@latest",
```

Comment:
ml_client.Environment.create_or_update returns an instance of environment. Please pass it to the common job instead of trying resolve the latest label based on the assumption it was created before.

Reply:
Updated!